docs/userGuide/syntax/icons.md

```diff
@@ -8,19 +8,18 @@ MarkBind supports using Font Icons provided by Font Awesome, Glyphicons and GitH
 <span id="tip_body">
 The advantage of font icons over emojis is font icons can be _styled_ to fit your needs. e.g.,
 * emoji: <span style="color: purple">Don't judge the :book: by its cover! :-1:</span>
-* font icons: <span style="color: purple">Don't judge the :fas-book: by its cover! {{ icon_dislike }}</span>
+* font icons: <span style="color: purple">Don't judge the :fa-solid-book: by its cover! {{ icon_dislike }}</span>
 </span>
 </include>
 
 ###### Using Font Awesome Icons
 1. Decide which icon you want to use from the [list of available icons](https://fontawesome.com/icons?d=gallery&m=free).
 1. Construct the MarkBind name for the selected icon by adding the _type prefix_.
    Note: Font Awesome has three different styles for their icons, each with their own type prefix. Here is an example from each type:
-   * _Solid_ (prefix: `fas-`) e.g., :fas-file-code: (actual name `file-code`, MarkBind name `fas-file-code`)
-   * _Regular_ (prefix: `far-`) e.g., :far-file-code: (actual name `file-code`, MarkBind name `far-file-code`)
```

Comment:
What happens to regular? line 18 still says 3 types.

Reply:
v6 has moved the regular icons to the pro version:
<img width="654" alt="image" src="https://github.com/MarkBind/markbind/assets/30100720/840bc68e-91f6-47d8-badb-8c126d6bbffb">

However, due to the icon aliasing in v6, we can still use v5 regular icons:
```
:fa-solid-house:
:far-file-code:
:fa-brands-facebook:
:fab-facebook:
```
<img width="110" alt="image" src="https://github.com/MarkBind/markbind/assets/30100720/b6c289f1-6c4a-4af2-a9b8-4a1a6da8c11d">

<img width="612" alt="image" src="https://github.com/MarkBind/markbind/assets/30100720/c0eef6dd-1442-42ef-93a4-bee1c39cb8ef">

Reply:
We should mention this in our docs/release notes. Could you help add in/draft such relevant info?
If a user who's currently using font-awesome icons and look at the docs
- should be able to find out what significant changes are made, i.e. syntax has changed, some icons have been renamed although the old ones still work
- be encouraged to update their naming to avoid breaking changes in future updates